polynote-server/src/main/scala/polynote/server/KernelLaunching.scala

```diff
@@ -1,7 +1,7 @@
 package polynote.server
 
 import cats.effect.{ContextShift, IO, Timer}
-import polynote.kernel.dependency.{CoursierFetcher, DependencyFetcher}
+import polynote.kernel.dependency.{CoursierFetcher, DependencyFetcher, IvyFetcher}
```

Comment:
Probably don't need this import.

There's currently no way of actually selecting the ivy fetcher. I'll probably delete it sometime soon, maybe after we get feedback on whether the new coursier fetcher works reliably.